File: src/main/java/de/maggiwuerze/xdccloader/events/bot/TargetBotUpdateEvent.java
package de.maggiwuerze.xdccloader.events.bot;

import de.maggiwuerze.xdccloader.events.EntityUpdateEvent;
import de.maggiwuerze.xdccloader.model.entity.Bot;

public class TargetBotUpdateEvent extends EntityUpdateEvent<Bot> {

	public TargetBotUpdateEvent(Object source, Bot bot) {
		super(source, bot);
	}
}

File: src/main/kotlin/de/maggiwuerze/xdccwebloader/events/bot/TargetBotUpdateEvent.kt
package de.maggiwuerze.xdccwebloader.events.bot

import de.maggiwuerze.xdccwebloader.events.EntityUpdateEvent
import de.maggiwuerze.xdccwebloader.model.entity.Bot

class TargetBotUpdateEvent(source: Any, bot: Bot) : EntityUpdateEvent<Bot>(source, bot)